config/environments/production.rb

```diff
@@ -93,4 +93,8 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.default_url_options = { host: 'https://cites-trade-tool.speciesplus.net/' }
```

Comment:
Maybe we can store the host in the `secrets.yml` file ?!

Reply:
Ah good point! I'll change that for staging as well :) Thanks!

Reply:
@lucacug That's done now!